src/views/Returns.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-menu-button slot="start" />
+        <ion-title>{{ $t("Returns") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content>
+      <main>
+        <ion-searchbar :placeholder="$t('Scan ASN to start receiving')"/>
+  
+        <ReturnListItem v-for="returnShipment in returns" :key="returnShipment.shipmentId" :returnShipment="returnShipment" />
+
+        <div class="load-more-action ion-text-center">
+          <ion-button fill="outline" color="dark" @click="loadMoreReturns()">
+            <ion-icon :icon="cloudDownloadOutline" slot="start" />
+            {{ $t("Load more returns") }}
+          </ion-button>
+        </div>
+      </main>
+    </ion-content>
+  </ion-page>
+</template>
+  
+<script lang="ts">
+  import { IonButton, IonContent, IonHeader, IonIcon, IonMenuButton, IonPage, IonSearchbar, IonTitle, IonToolbar } from '@ionic/vue';
+  import { cloudDownloadOutline } from 'ionicons/icons'
+  import { defineComponent } from 'vue'
+  import { mapGetters, useStore } from 'vuex'
+  import ReturnListItem from '@/components/ReturnListItem.vue'
+  
+  export default defineComponent({
+    name: "Returns",
+    components: {
+      IonButton,
+      IonContent,
+      IonHeader,
+      IonIcon,
+      IonMenuButton,
+      IonSearchbar,
+      IonPage,
+      IonTitle,
+      IonToolbar,
+      ReturnListItem
+    },
+    computed: {
+      ...mapGetters({
+        returns: 'return/getReturns',
+        user: 'user/getCurrentFacility'
+      })
+    },
+    mounted () {
+      this.getReturns();
+    },
+    methods: {
+      selectSearchBarText(event: any) {
+        event.target.getInputElement().then((element: any) => {
+          element.select();
+        })
+      },
+      async getReturns(vSize?: any, vIndex?: any) {
+        const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+        const viewIndex = vIndex ? vIndex : 0;
+        const payload = {
+          "inputFields": {
+            "destinationFacilityId": this.user.facilityId,
+            "statusId": "PURCH_SHIP_SHIPPED",
+            "shipmentTypeId_fld0_value": "INCOMING_SHIPMENT",
```

Comment:
Add single condition for shipmentTypeId as SALES_RETURN

Reply:
Done sir.